src/main/java/org/jabref/gui/desktop/JabRefDesktop.java

```diff
@@ -104,6 +104,8 @@ public static void openExternalViewer(BibDatabaseContext databaseContext,
 
         if (StandardField.URL == field) {
             openBrowser(link);
+        } else if (StandardField.ISBN == field) {
+          openIsbn(link);
```

Comment:
Since the code is similar to the DOI code. Please move after line 85 and add a `return` statement. This way, the code is really similar to the DOI code.

Reply:
addressed